### PR TITLE
Testing the repository classes

### PR DESCRIPTION
**Pull request by ha2san**
I test three of the four repository classes

### CODE REVIEW

**Comment by ha2san** [comment]
I testes some repo classes. I couldn't do better than that

File: app/src/androidTest/java/com/github/brugapp/brug/fake/FirebaseFakeHelper.kt [hunk 1]
```diff
@@ -19,9 +19,9 @@ class FirebaseFakeHelper {
         } catch (e: IllegalStateException) {
             // Already correct emulator
         }
-        val settings = FirebaseFirestoreSettings.Builder()
-            .setPersistenceEnabled(false)
-            .build()
+        //val settings = FirebaseFirestoreSettings.Builder()
+        //    .setPersistenceEnabled(false)
+        //    .build()
         //firestore.firestoreSettings = settings
```
Comment (hamzahass) [inline app/src/androidTest/java/com/github/brugapp/brug/fake/FirebaseFakeHelper.kt:25]: Are these comments still necessary ?

Comment (hamzahass) [inline app/src/androidTest/java/com/github/brugapp/brug/fake/FirebaseFakeHelper.kt:25]: Same for here ?

**Review by hamzahass — APPROVED** [review]
Thanks for adding tests for these Repositories. Consider the comments I left bellow.

**Review by omarelmalki — APPROVED** [review]
Great job on the new tests.

File: app/src/androidTest/java/com/github/brugapp/brug/fake/FirebaseFakeHelper.kt (hunk 1 above)
Comment (ha2san) [reply app/src/androidTest/java/com/github/brugapp/brug/fake/FirebaseFakeHelper.kt:25]: Nope

Comment (ha2san) [reply app/src/androidTest/java/com/github/brugapp/brug/fake/FirebaseFakeHelper.kt:25]: Same

**Pull request merged**